hack/gen-python-sdk/post_gen.py

```diff
@@ -164,6 +165,42 @@ def update_python_sdk(src, dest, versions=('v1beta1')):
     # Clear working dictionary
     shutil.rmtree(src)
 
+def generate_sphinx_docs(src, dest):
+    
+    project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+
+    docs_dir = os.path.join(project_root, 'docs', 'sdkdocs')
```

Comment:
Currently we are usinhardcoded paths, does it makes sense to use environment variables instead for better configuration? @andreyvelich let me know your thoughts

Reply:
+1. Having environment variable would be Better for easy configuration management.